src/helics/shared_api_library/ValueFederate.h

```diff
@@ -492,7 +492,7 @@ HELICS_EXPORT int helicsInputGetRawValueSize(helics_input ipt);
  * @return Raw string data.
  * @endPythonOnly
  */
-HELICS_EXPORT void helicsInputGetRawValue(helics_input ipt, void* data, int maxDatalen, int* actualSize, helics_error* err);
+HELICS_EXPORT void helicsInputGetRawValue(helics_input ipt, void* data, int maxDataLength, int* actualSize, helics_error* err);
```

Comment:
Same here,  this change requires and additional change in the swig .i files

Reply:
Resolved in 7e460d0.